central/networkgraph/service/service_impl_postgres_test.go

```diff
@@ -129,6 +131,123 @@ func externalFlow(deployment *storage.Deployment, entity *storage.NetworkEntity,
 	}
 }
 
+func normalizeTimestamps(graph *v1.NetworkGraph, timeNow *timestamppb.Timestamp) {
+	for _, node := range graph.Nodes {
+		for _, edgeBundle := range node.OutEdges {
+			for _, edge := range edgeBundle.Properties {
+				if edge.LastActiveTimestamp != nil {
+					edge.LastActiveTimestamp = timeNow
+				}
+			}
+		}
+	}
+}
+
+func (s *networkGraphServiceSuite) TestGetNetworkGraph() {
+	ctx := sac.WithGlobalAccessScopeChecker(
+		context.Background(),
+		sac.AllowFixedScopes(
+			sac.AccessModeScopeKeys(storage.Access_READ_ACCESS),
+			sac.ResourceScopeKeys(resources.Deployment, resources.NetworkGraph),
+		),
+	)
+
+	globalWriteAccessCtx := sac.WithGlobalAccessScopeChecker(context.Background(),
+		sac.AllowFixedScopes(
+			sac.AccessModeScopeKeys(storage.Access_READ_ACCESS, storage.Access_READ_WRITE_ACCESS),
+			sac.ResourceScopeKeys(resources.NetworkGraph, resources.Deployment)))
+
+	entityID1, _ := externalsrcs.NewClusterScopedID(testCluster, "192.168.1.1/32")
+	entityID2, _ := externalsrcs.NewClusterScopedID(testCluster, "10.0.0.2/32")
+	entityID3, _ := externalsrcs.NewClusterScopedID(testCluster, "1.1.1.1/32")
+
+	entityIDs := []string{
+		entityID1.String(), entityID2.String(), entityID3.String(),
+	}
```

Comment:
[nit]

since we're just using the list - we might as well combine:
```suggestion
	entityIDs := []string{
			externalsrcs.NewClusterScopedID(testCluster, "192.168.1.1/32"),
	        externalsrcs.NewClusterScopedID(testCluster, "10.0.0.2/32"),
	        externalsrcs.NewClusterScopedID(testCluster, "1.1.1.1/32"),
	}
```